src/Navcoin/CommunityFund/Entity/Proposal.php

```diff
@@ -257,4 +257,9 @@ public function getVotingCycle(): int
     {
         return $this->votingCycle;
     }
-}
\ No newline at end of file
+
+    public function getIsSuper(): bool
+    {
+        return $this->version == 106;
```

Comment:
this should be checked `$this->version & 1<<6`

Reply:
Updated as per suggestion, is there a reason why it was working with checking it against an INT?